libhdf5/H5FDhttp.h

```diff
@@ -30,7 +29,12 @@
 
 #include "H5Ipublic.h"
 
+#if H5_VERSION_GE(1,14,0)
+#define H5_VFD_HTTP     ((H5FD_class_value_t)(H5_VFD_MAX - 2))
```

Comment:
it would be more prudent to reserve a dedicated VFD. Cf https://portal.hdfgroup.org/pages/viewpage.action?pageId=74188097

Reply:
Good idea. Will do so ASAP.